docs/intro.md

```diff
@@ -101,7 +101,7 @@ Slack is good for that, too: [join slack](https://greatexpectations.io/slack).
 If you have bugfix or feature request, please [upvote an existing issue](https://github.com/great-expectations/great_expectations/issues) or [open a new issue](https://github.com/great-expectations/great_expectations/issues/new) on GitHub and we’ll see what we can do.
 
 #### Contribute code or documentation
-We welcome contributions to Great Expectations. Please start with our [Contributing guide](/docs/contributing/contributing) and don’t be shy with questions!
+We welcome contributions to Great Expectations. Please start with our [Contributing guide](../contributing.md) and don’t be shy with questions!
```

Comment:
This should be `./contributing/contributing.md`

Reply:
I made the fix in new commit.